maximbircu/devtools-library
# Add iOS toggle tool implementation

Closes: #15


### What has been done
 1. Add iOS implementation of toggle tool
 2. Add intercepting of view from storyboard for replacing it with devtool with specific devtool element

### Screenshots
iOS |
:-: |
![image](https://user-images.githubusercontent.com/23414370/109437531-e8a24e00-7a2d-11eb-92ef-e846cc35d418.png)|

### How to test
1. Run `./gradlew assemble ` for generating iOS fat framework and placing it in directory with iOS wrapper framework
2. Open iOS wrapper framework and check if it builds without erros
3. Open iOS sample app project and check if it builds without errors and toggle view is presented at app start
